src/java.base/share/classes/java/util/zip/ZipError.java

```diff
@@ -28,9 +28,12 @@
 /**
  * Signals that an unrecoverable error has occurred.
  *
+ * @deprecated ZipError is no longer used and is obsolete.
```

Comment:
I agree ZipError should be deprecated. Can the JBS issue and PR title be updated as they still have "for removal".

Reply:
Isn't this deprecation intended for removal? Or do you mean this `@deprecated` tag should note that this class will be removed in a future release?

Reply:
Alan,

This is probably better discussed in the CSR. Please see the latest comments there.

I think we're still waiting for concensus/conclusion in the CSR on whether the deprecation of the class itself should be for removal or not.

Joe suggested that the class be marked deprecated, not for removal, and that the constructor be marked deprecated, for removal. He asked for input on this suggestion, which unfortunately took me a while. 

The latest comment from me and @liach made a case for deprecating the class for removal. We are waiting for input from Joe on this.

Once the CSR concludes on a solution, I'll of course update the CSR and PR accordingly.

Thanks.